### PR TITLE
Use https-proxy-agent instead of tunnel. Fixes #53

### LINKED ISSUE #53: http proxy is not working using node 0.12.7
Hi,
Unfortunately, updating the nodejs version, the librato backend is not working anymore.

More precisely, we are getting the following excpetion 

```
_http_client.js:73
    throw new TypeError('Request path contains unescaped characters.');
          ^
TypeError: Request path contains unescaped characters.
    at new ClientRequest (_http_client.js:73:11)
    at TunnelingAgent.exports.request (http.js:49:10)
    at TunnelingAgent.createSocket (/opt/statsd/node_modules/tunnel/lib/tunnel.js:116:25)
    at TunnelingAgent.createSecureSocket [as createSocket] (/opt/emarsys/statsd/node_modules/tunnel/lib/tunnel.js:188:41)
    at TunnelingAgent.addRequest (/opt/statsd/node_modules/tunnel/lib/tunnel.js:80:8)
    at new ClientRequest (_http_client.js:154:16)
    at Object.exports.request (http.js:49:10)
    at Object.exports.request (https.js:136:15)
    at post_payload (/opt/statsd/node_modules/statsd-librato-backend/lib/librato.js:64:18)
    at post_metrics (/opt/statsd/node_modules/statsd-librato-backend/lib/librato.js:149:3)
```

As you can see is a problem in the tunnel library, I was wondering If it would make sense to use another library.

### PR DESCRIPTION
**Pull request by basetta**
Used the module **https-proxy-agent** instead of tunnel seeing that with node 0.12.7 is not working. 

Tested on our environment and it is working. Keeping on mind that I am not a node expert :D

### CODE REVIEW

File: lib/librato.js [hunk 1]
```diff
@@ -60,7 +61,7 @@ var globalPrefix = "";
 
 var post_payload = function(options, proto, payload, retry)
 {
-  var req = proto.request(options, function(res) {
+ var req = proto.request(options, function(res) {
```
Comment (mheffner) [inline lib/librato.js]: Seems like original indent was fine?

**Comment by mheffner** [comment]
:+1:

File: lib/librato.js (hunk 1 above)
Comment (basetta) [inline lib/librato.js]: you are right. Fixed :)

**Comment by mheffner** [comment]
Merged, thanks!

**Pull request merged**